taglibs/async/await-tag.js

```diff
@@ -23,6 +23,10 @@ function promiseToCallback(promise, callback, thisObj) {
             finalPromise = finalPromise.fail(function(err) {
                 callback(err);
             });
+        } else {
+            finalPromise = finalPromise.then(void 0, function(err) {
```

Comment:
I can certainly change this if desired, all ES5+ environments will handle both identically (in fact I vaguely remember reading somewhere that `undefined` gets compiled into `void 0` for performance reasons).